README.md

```diff
@@ -10,4 +10,13 @@ How to run :
 Output : 
    - Go to the logs folder and look for .csv files with recommendations. CSV files are named after the index order of the thanos url.
 
+Run as a Kubernetes Job :
+   - Have the ./charts/job.yaml file updated with your image pull secret
+   - Have the ./charts/configmap.yaml file updated with your thanos urls and respective tokens
+   - Have the ./charts/pvc.yaml file updated with your storageclass name
+   - run `oc apply -f ./charts`
+
+Output :
+   - Go to the current date folder in the PVC mount and look for .csv files with recommendations. CSV files are named after the index order of the thanos url.
+
```

Comment:
Content wise, very clear. Just a few nit points.

1. Should we change `How to run :` to `Run it from your laptop` (or something) that contrasts wioth `Run as a Kubernetes Job:`
2. Should we make the above 2 bold.